src/clj/tasks/load_test.clj

```diff
@@ -0,0 +1,121 @@
+
+(defn add-test-users [maxUsers]
+  (webdb/connect)
+  (when (not (mc/find-one-as-map db "users" {:username "TestCorp"}))
+    (mc/insert db "users"
+                    {:username         "TestCorp"
+                      :email            "TestCorp@mailinator.com"
+                      :password         (password/encrypt "password")
+                      :isadmin          false
+                      :options          {}}))
+
+  (when (not (mc/find-one-as-map db "users" {:username "TestRunner"}))
+    (mc/insert db "users"
+                    {:username         "TestRunner"
+                      :email            "TestRunner@mailinator.com"
+                      :password         (password/encrypt "password")
+                      :isadmin          false
+                      :options          {}}))
+  (doall
+    (pmap
+      (fn [n]
+        (when (not (mc/find-one-as-map db "users" {:username (str "TestUser" n)}))
+          (mc/insert db
+                    "users"
+                    {:username         (str "TestUser" n)
+                      :email            (str "TestUser" n "@mailinator.com")
+                      :password         (password/encrypt "password")
+                      :isadmin          false
+                      :options          {}})))
+      (range maxUsers))))
+
+(defn login
+  [username password]
+  (let [options {:form-params {:username username :password password}}
+      {:keys [status error headers body]} @(http/post "http://localhost:1042/login" options)]
+  (if (or error (= 401 status))
+    (println "Failed, exception is " (or error status))
+    (:set-cookie headers))))
+
+(defn create-game [maxUsers]
+  (def corp-client-id (uuid/to-string (uuid/v1)))
+  (def runner-client-id (uuid/to-string (uuid/v1)))
+  (safe-println "Login with test users")
+  (def corp-socket (ws/connect
+        (str "ws://localhost:1042/ws?client-id=" corp-client-id)
+        ; :on-receive #(safe-println 'received %)
+        :on-error #(safe-println "corp error" %)
+        :on-connect (fn [n] (safe-println "Corp Connected"))
+        :on-close (fn [x y] (safe-println "Corp Disconnected"))
+        :headers {"Cookie" (login "TestCorp" "password")}
+        ))
+  (def runner-socket (ws/connect
+        (str "ws://localhost:1042/ws?client-id=" runner-client-id)
+        ; :on-receive #(safe-println 'received %)
+        :on-error #(safe-println "runner error" %)
+        :on-connect (fn [n] (safe-println "Runner Connected"))
+        :on-close (fn [x y] (safe-println "Runner Disconnected"))
+        :headers {"Cookie" (login "TestRunner" "password")}))
+  (safe-println "Create lobby")
+  (handle-lobby-create {:ring-req {:user {:username "TestCorp"}}
+                        :client-id corp-client-id
+                        :?data {:title "Performance Game" :format "standard" :allow-spectator true :spectatorhands false :password "" :room "casual" :side "Corp" :options {}}})
```

Comment:
This will be more readable if you put each key-value pair on a new line, like above.